app/scripts/controllers/transactions/index.test.js

```diff
@@ -1294,7 +1299,7 @@ describe('Transaction Controller', function () {
     });
 
     it('should add multiple transactions when called with different actionId', async function () {
-      const txMeta = await txController.addUnapprovedTransaction({
+      const txMeta = await txController.addUnapprovedTransaction(undefined, {
         from: selectedAddress,
         to: recipientAddress,
       });
```

Comment:
Will be useful to have a test case where `method` is defined.

Reply:
Fixed here: 352f1d9054aefd84dd03b992873db4bf61cebc39

Reply:
Will be nice to have a test case that tests the functionality added by `txMethodType` argument. That is `opts.securityProviderRequest` should be called.

Reply:
Fixed here: c082a3c47bd15eb384299535d71407032be752e8